src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  size: 12px;
+  font-style: normal;
+  line-height: 14px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  background-color: gray;
+}
+
+.header {
+  background-color: white;
+  width: 1200px;
+  height: 60px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 20px;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 60px;
+  margin: 40px;
```

Comment:
The `.logo` class has both `top` and `left` properties which won't have any effect unless the position is set to `absolute`, `relative`, or `fixed`. Also, the `margin: 40px;` seems excessive, especially if this is meant to be a logo within a header. Consider adjusting the positioning and margin to better reflect the design requirements.